Point to VM instructions when changing SLES to SUMA (bsc#1161836) (#58)
uyuni-project/uyuni-docs: modules/installation/pages/install-server-unified.adoc

For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].

[NOTE]
====

> To change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber} you need to follow the instructions at xref:install-vm.adoc[Install Server in a Virtual Machine with JeOS].

Reviewer: ```suggestion
To change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}, follow the instructions at xref:installation:install-vm.adoc[].
```
